### ISSUE TITLE
Incorrect "this" reference in button.js

### ISSUE THREAD

**Issue by EvanBalster** [Jermolene/TiddlyWiki5#3078]
While working on a widget based on button I noticed this small aberration.  An improper reference to "this" when "self" would be correct:

```
70		if(self.invokeActions(this,event)) {
71			handled = true;
72		}
```

**Comment by pmario** [comment]
I think it's ok. ... `self.invokeActions()` is defined in widget.js. It's signature is: 

```
Widget.prototype.invokeActions = function(triggeringWidget,event) {
```

Where `triggeringWidget` imo expects the DOM element, that triggered the action. 

Whereas `self` is a reference to the button widget  object.

**Comment by pmario** [comment]
uups, You are probably right. I'm single-stepping through the code atm. ... 

What's the problem, that you face?

**Comment by pmario** [comment]
Can you post the `wiki-text`that you use for testing?

**Comment by EvanBalster** [comment]
No problems with my wiki — and hence no wikitext.  Just something incorrect I spotted in the code while working on a new widget of my own.  invokeActions is supposed to receive a triggeringWidget, which isn't done correctly here.

**Comment by pmario** [comment]
>No problems with my wiki — and hence no wikitext. Just something incorrect I spotted in the code while working on a new widget of my own. invokeActions is supposed to receive a triggeringWidget, which isn't done correctly here.

With my single stepping, I came to the same conclusion. So seems to be a bug

**Comment by Jermolene** [comment]
Fixed in #3185
